src/item.py

```diff
@@ -0,0 +1,4 @@
+class Item:
+  def __init__(self, name, description):
+    self.name = name,
+    self.description = description
```

Comment:
You'll want to add on_take and on_drop methods to this class when you get there.